flink-connectors/flink-sql-connector-hive-3.1.3/pom.xml

```diff
@@ -142,18 +149,19 @@ under the License.
 								<filter>
 									<artifact>org.apache.hive:hive-exec</artifact>
 									<excludes>
-										<exclude>org/apache/hadoop/hive/conf/HiveConf.class</exclude>
-										<exclude>org/apache/hadoop/hive/metastore/HiveMetaStoreClient.class</exclude>
+										<exclude>avro/shaded/**</exclude>
```

Comment:
Why did you add this?

Reply:
Adding `<exclude>avro/shaded/**</exclude>` removes the following warning in compilation of `flink-sql-connector-hive-3.1.3`

```
[WARNING] hive-exec-3.1.3.jar, avro-1.8.2.jar define 799 overlapping classes: 
[WARNING]   - avro.shaded.com.google.common.collect.SingletonImmutableList
[WARNING]   - avro.shaded.com.google.common.collect.Iterables$15
[WARNING]   - avro.shaded.com.google.common.collect.Sets$PowerSet$1$1
[WARNING]   - avro.shaded.com.google.common.collect.RegularImmutableMap
[WARNING]   - avro.shaded.com.google.common.collect.Synchronized$SynchronizedSortedSet
[WARNING]   - avro.shaded.com.google.common.collect.Synchronized$SynchronizedObject
[WARNING]   - avro.shaded.com.google.common.cache.LocalCache$Strength
[WARNING]   - avro.shaded.com.google.common.collect.ForwardingMultiset
[WARNING]   - avro.shaded.com.google.common.collect.MapMakerInternalMap$WeakExpirableEntry
[WARNING]   - avro.shaded.com.google.common.collect.Collections2$FilteredCollection$1
[WARNING]   - 789 more...
[WARNING] maven-shade-plugin has detected that some class files are
[WARNING] present in two or more JARs. When this happens, only one
[WARNING] single version of the class is copied to the uber jar.
[WARNING] Usually this is not harmful and you can skip these warnings,
[WARNING] otherwise try to manually exclude artifacts based on
[WARNING] mvn dependency:tree -Ddetail=true and the above output.
[WARNING] See http://maven.apache.org/plugins/maven-shade-plugin/

```